src/ThirdParty/tinyxml/tinyxml.h

```diff
@@ -276,7 +276,7 @@ class TiXmlBase
 		TIXML_ERROR_PARSING_COMMENT,
 		TIXML_ERROR_PARSING_DECLARATION,
 		TIXML_ERROR_DOCUMENT_EMPTY,
-		TIXML_ERROR_EMBEDDED_NULL,
+		TIXML_ERROR_EMBEDDED_nullptr,
```

Comment:
```suggestion
		TIXML_ERROR_EMBEDDED_NULL,
```

Another mistaken replacement